packages/core/package.json

```diff
@@ -46,7 +46,6 @@
     "client-oauth2": "^4.2.5",
     "cron": "~1.7.2",
     "crypto-js": "~4.1.1",
-    "digest-header": "^0.0.1",
```

Comment:
We can remove the import for this lib, if it works without it.

Reply:
already did